README.md

```diff
@@ -23,7 +23,7 @@ Create the virtualenv:
 
 Install the requirements:
 
-    pip install -r requirements.txt
+    sudo pip install -r requirements.txt
```

Comment:
Will you at least merge in the versioning changes? They have nothing to do with virtualenv. But thanks for telling me so I didn't make the changes.